benchmarks/benchmark_base.py

```diff
@@ -0,0 +1,453 @@
+import re
+from copy import deepcopy
+from os.path import dirname, realpath, join
+from pathlib import Path
+from tempfile import mkstemp
+
+import astropy.units as u
+import numpy as np
+import pandas as pd
+from numba import njit
+
+from tardis.io.atom_data import AtomData
+from tardis.io.configuration import config_reader
+from tardis.io.configuration.config_reader import Configuration
+from tardis.io.util import yaml_load_file, YAMLLoader, HDFWriterMixin
+from tardis.model import SimulationState
+from tardis.montecarlo import NumbaModel, opacity_state_initialize
+from tardis.montecarlo.montecarlo_numba import RPacket
+from tardis.montecarlo.montecarlo_numba.packet_collections import (
+    VPacketCollection,
+)
+from tardis.simulation import Simulation
+from tardis.tests.fixtures.atom_data import DEFAULT_ATOM_DATA_UUID
+
+
+class BenchmarkBase:
+    timeout = 369
+
+    @staticmethod
+    def split_path(path):
+        return path.split('/')
```

Comment:
Okay. The `Path()` object has a `.parts` method that should produce the same result without a custom method like this, see e.g. second answer here https://stackoverflow.com/questions/10149263/extract-a-part-of-the-filepath-a-directory-in-python

Reply:
Done in the commit: 6e0fd31fb6f1209fd2898797497bfb40f34c8c03.

I removed the function `split_path` because it only has one usage, when I created this code I assumed that it will be used more, but it is not true.